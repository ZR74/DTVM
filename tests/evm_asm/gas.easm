// gas: remaining gas (after this instruction)
// GASLIMIT - BASIC_EXECUTION_COST - ACTUAL_COST = 0xFFFFFFFFFFFF - 2100 - 2 = 0xFFFFFFFFADF5
GAS
PUSH1 0x00
MSTORE
PUSH1 0x20
PUSH1 0x00
RETURN
