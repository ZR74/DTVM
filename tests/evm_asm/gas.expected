status: success
error_code: 0
stack: []
memory: '0000000000000000000000000000000000000000000000000000FFFFFFFFADF5'
storage: {}
transient_storage: {}
return: '0000000000000000000000000000000000000000000000000000FFFFFFFFADF5'
events: []
